openlibrary/templates/account/borrow.html

```diff
@@ -37,6 +37,10 @@
 th.status {
     width: 150px;
 }
+.LoanReadForm {
```

Comment:
It would be useful to move this new style (and all the other styles here) to an import named "BorrowForm.less" inside static/css/page-user.less

All these rules should be scoped to `#borrowTable` to ensure they don't apply to other elements.